### DISCUSSION TITLE
how to "local boot" linux on dualboot system?

### DISCUSSION THREAD

**Discussion by Ryder17z** [ventoy/Ventoy#2878] [General]
Usecase:
* Some form of dualboot of windows and linux 

* No matter what "local boot" option is selected will always load windows from my brief testing.

There should be a way to do this but i don't know how.

**Comment by randompersononinternet69** [comment]
It would need to detect the kernel and initrd, and get the kernel parameters from it's grub.cfg file (if it exists)
